frontend/packages/console-app/src/components/console-operator/ConsolePluginManifestPage.tsx

```diff
@@ -14,10 +16,25 @@ export const ConsolePluginManifestPage: React.FC<PageComponentProps> = ({ obj })
     pluginName,
   ]);
 
+  const manifestJson = React.useMemo(() => {
+    return pluginManifest ? JSON.stringify(pluginManifest, null, 2) : '';
+  }, [pluginManifest]);
+
   return (
     <PaneBody>
       {pluginManifest ? (
-        <CopyToClipboard value={JSON.stringify(pluginManifest, null, 2)} />
+        <CodeEditor
+          value={manifestJson}
```

Comment:
this will add a thingy that says "json"

<img width="1635" height="1010" alt="image" src="https://github.com/user-attachments/assets/b7ec38a6-44f5-4cf5-b3fe-d118c5c34ff1" />

```suggestion
          value={manifestJson}
          isLanguageLabelVisible
```